Wrappers/Python/test/test_SIRF.py

```diff
@@ -114,7 +116,69 @@ def test_BlockDataContainer_with_SIRF_DataContainer_add(self):
         bdc = BlockDataContainer(image1, image2)
 
         self.assertBlockDataContainerEqual(bdc , bdc1)
-    
+
+    @unittest.skipUnless(has_sirf, "Has SIRF")
+    def test_GradientSIRF_3D_pseudo_geometries(self):
+
+        shutil.rmtree('working_folder/thorax_single_slice',True)
```

Comment:
I have added the `thorax_single_slice`. However, I tried to test it locally but there is no `thorax_single_slice` folder. Only `brain` and `mMR`. @paskino  any ideas?